lib/RenderApp/Controller/IO.pm

```diff
@@ -82,7 +85,7 @@ sub search {
   my @targetArray = split /\//, $target;
 
   local $File::Find::skip_pattern = qr/^\./; #skip any hidden folders
-  my @sources = ('private/', 'webwork-open-problem-library/Library/', 'webwork-open-problem-library/Contrib/');
+  my @sources = ('private/', 'webwork-open-problem-library/OpenProblemLibrary/', 'webwork-open-problem-library/Contrib/');
```

Comment:
Why are we changing this?

Reply:
because the old path is actually invalid -- `Library` is the 'abbreviation' that is used by default in ww2 (as a symlink to the 'real' path given by the change made here)